docs/modules/ROOT/pages/references/services-vshn.adoc

```diff
@@ -129,3 +129,44 @@ type:: bool
 default:: `true`
 
 Whether to enable NetworkPolicy in the Redis instance namespace. This allows to establish connections from claim's namespace to the Redis instance.
+
+=== `redis.plans`
+type:: dict
+
+A dict of plans for Redis by VSHN.
+
+The key is the name of the plan.
+You can configure the CPU request through `size.cpu`, memory through `size.memory`, and disk size through `size.disk`.
+You can also set a node selector through `scheduling.nodeSelector`.
+
+There is also the option to specify a note in `note`, which will be added to the description of the field in the CRD.
+
+.Examples
+[source,yaml]
+----
+plans:
+  standard-2:
+    size:
+      cpu: "500m"
+      memory: "2Gi"
+      disk: 16Gi
+  standard-4:
+    size:
+      cpu: "1"
+      memory: "4Gi"
+      disk: 16Gi
+  standard-8:
+    enabled: false
+  plus-4:
+    size: ${appcat:services:vshn:redis:plans:standard-4:size}
+    scheduling:
+      nodeSelector:
+        appuio.io/node-class: "plus"
+    note: "Will be scheduled on APPUiO Cloud plus nodes"
```

Comment:
I'd add https://docs.appuio.cloud/user/references/node-classes.html this link somewhere here, so it's easy and fast to find specifics of nodes, not everyone must be aware what "plus" means